### DISCUSSION TITLE
My Request gets rejected and getting the error that I have not completed my Profile

### DISCUSSION THREAD

**Discussion by jhinga-lala-ho-ho** [community/community#54628] [GitHub Education]
### Hi everyone,

I have tried multiple times but I am getting the same error that My Profile is not completed.
How to fix this?

**Comment by samus-aran** [comment]
Hi @Robot508 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
